todo_app/e2e_test.py

```diff
@@ -3,24 +3,52 @@
 import requests
 from threading import Thread
 from selenium import webdriver
-#from todo_app.app import create_app
-#os.path.dirname(r'todo_app\app.py')
+from flask import Flask
+import os
 from todo_app.app import create_app
+from dotenv import load_dotenv,find_dotenv
+
+
 
 
-@pytest.fixture(scope='module')
 def create_trello_board():
-    board_id = create_trello_board()
-    os.environ['TOID'] = board_id
-    application = create_app()
-    return application
+    
+    key = os.environ['KEY']
+    token = os.environ['TOKEN']
+    board_response = requests.post(f'https://api.trello.com/1/boards/?name=test&key={key}&token={token}')
+    board_id = board_response.json()["id"]
+    response =  requests.get(f'https://api.trello.com/1/boards/{board_id}/lists?key={key}&token={token}')
+    
+	
+    lists = response.json()
+    
+    for list in lists:
+        
+        if list['name'] == "TODO":
+            os.environ['TOID'] = list['id']
+        elif list['name'] == "DOING":
+            os.environ['DOING'] = list['id']
```

Comment:
The environment variable you check for in the code is `DOINGID` rather than `DOING`

Reply:
Thank you, Updated